samples/eshop/services/webstatus.bicep

```diff
@@ -25,25 +24,25 @@ resource webstatus 'Applications.Core/containers@2023-10-01-preview' = {
         ASPNETCORE_ENVIRONMENT: 'Development'
         ASPNETCORE_URLS: 'http://0.0.0.0:80'
         HealthChecksUI__HealthChecks__0__Name: 'WebMVC HTTP Check'
-        HealthChecksUI__HealthChecks__0__Uri: 'http://webmvc:5100/hc'
+        HealthChecksUI__HealthChecks__0__Uri: 'http://webmvc:5100/liveness'
         HealthChecksUI__HealthChecks__1__Name: 'WebSPA HTTP Check'
-        HealthChecksUI__HealthChecks__1__Uri: 'http://web-spa:5104/hc'
+        HealthChecksUI__HealthChecks__1__Uri: 'http://web-spa:5104/liveness'
         HealthChecksUI__HealthChecks__2__Name: 'Web Shopping Aggregator GW HTTP Check'
-        HealthChecksUI__HealthChecks__2__Uri: 'http://webshoppingagg:5121/hc'
+        HealthChecksUI__HealthChecks__2__Uri: 'http://webshoppingagg:5121/liveness'
         HealthChecksUI__HealthChecks__4__Name: 'Ordering HTTP Check'
-        HealthChecksUI__HealthChecks__4__Uri: 'http://ordering-api:5102/hc'
+        HealthChecksUI__HealthChecks__4__Uri: 'http://ordering-api:5102/liveness'
         HealthChecksUI__HealthChecks__5__Name: 'Basket HTTP Check'
-        HealthChecksUI__HealthChecks__5__Uri: 'http://basket-api:5103/hc'
+        HealthChecksUI__HealthChecks__5__Uri: 'http://basket-api:5103/liveness'
         HealthChecksUI__HealthChecks__6__Name: 'Catalog HTTP Check'
-        HealthChecksUI__HealthChecks__6__Uri: 'http://catalog-api/hc'
+        HealthChecksUI__HealthChecks__6__Uri: 'http://catalog-api/liveness'
         HealthChecksUI__HealthChecks__7__Name: 'Identity HTTP Check'
-        HealthChecksUI__HealthChecks__7__Uri: 'http://identity-api:5105/hc'
+        HealthChecksUI__HealthChecks__7__Uri: 'http://identity-api:5105/liveness'
         HealthChecksUI__HealthChecks__8__Name: 'Payments HTTP Check'
-        HealthChecksUI__HealthChecks__8__Uri: 'http://payment-api:5108/hc'
+        HealthChecksUI__HealthChecks__8__Uri: 'http://payment-api:5108/liveness'
         HealthChecksUI__HealthChecks__9__Name: 'Ordering SignalRHub HTTP Check'
-        HealthChecksUI__HealthChecks__9__Uri: 'http://ordering-signalrhub:5112/hc'
+        HealthChecksUI__HealthChecks__9__Uri: 'http://ordering-signalrhub:5112/liveness'
         HealthChecksUI__HealthChecks__10__Name: 'Ordering HTTP Background Check'
-        HealthChecksUI__HealthChecks__10__Uri: 'http://ordering-backgroundtasks:5111/hc'
+        HealthChecksUI__HealthChecks__10__Uri: 'http://ordering-backgroundtasks:5111/liveness'
```

Comment:
were we using the health check incorrectly before?

Reply:
The eshop code has a readiness check at `/hc` and a liveness check at `/liveness`. We had the readiness check hooked up as a liveness check